mycli/main.py

```diff
@@ -489,6 +497,18 @@ def _connect():
             self.echo(str(e), err=True, fg='red')
             exit(1)
 
+
+    def get_password_from_file(self, password_file):
+        password_from_file = None
+        if password_file:
+            if (os.path.isfile(password_file) or \
+                stat.S_ISFIFO(os.stat(password_file).st_mode)) \
```

Comment:
I think the linter will complain about indentations here. You can check with `python setup.py lint` (and fix with `python setup.py lint --fix`)

Reply:
fixed it manually but still ran lint with --fix which seems to have fixed a couple of other items